usr/lib/prime-indicator/prime-indicator

```diff
@@ -283,8 +255,8 @@ class Indicator:
         return "Force NVIDIA GPU to " + ("power OFF" if self.is_nvidia_on() else "power ON")
 
     def is_nvidia_on(self) -> bool:
-        out = subprocess.getoutput("cat /proc/acpi/bbswitch")
-        return out.lower().endswith("on")
+        out = subprocess.getoutput("cat /sys/kernel/debug/vgaswitcheroo/switch")
+        return "DIS: :Off" in out
```

Comment:
should be "not in", or should change "Off" to the opposite which I believe is "Pwr" but I need to confirm